libs/src/services/solana/SolanaWeb3Manager.ts

```diff
@@ -230,20 +231,30 @@ export class SolanaWeb3Manager {
     )
   }
 
-  async doesUserbankExist(sourceEthAddress?: string) {
-    const userbank = await this.deriveUserBank(sourceEthAddress)
+  async doesUserbankExist({
+    sourceEthAddress,
+    mint = DEFAULT_MINT
+  }: {
+    sourceEthAddress?: string
+    mint?: MintName
+  }) {
+    const userbank = await this.deriveUserBank({ sourceEthAddress, mint })
     const tokenAccount = await this.getTokenAccountInfo(userbank.toString())
     return !!tokenAccount
   }
 
   /**
    * Creates a solana bank account, either for optional `recipientEthAddress` or from the web3 provider's eth address
    */
-  async createUserBank(
-    feePayerOverride: string,
-    recipientEthAddress?: string,
-    mint: MintName = 'audio'
-  ) {
+  async createUserBank({
+    feePayerOverride,
+    sourceEthAddress,
```

Comment:
nit: comment lies

although I think if we're breaking things we could name this `ethAddress` rather than `sourceEthAddress`? Source feels like an unnecessary qualifier. totally dealer's choice though

Reply:
Oh good catch. I might leave it for now just so the name jives with the variables being passed in where it is used. Agreed that it's not the clearest description :-(

Reply:
Eh. Nevermind, it's not that many places. I shall update.